src/index.html

```diff
@@ -1,18 +1,64 @@
 <!doctype html>
 <html lang="en">
-  <head>
+  <head class="page">
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The `href` attribute is pointing to an SCSS file (`index.scss`). Browsers cannot directly interpret SCSS files, so you need to compile this SCSS file into a CSS file and update the `href` to point to the compiled CSS file (e.g., `index.css`).